cdc/processor/processor.go

```diff
@@ -1040,6 +1049,7 @@ func (p *processor) flushRedoLogMeta(ctx context.Context) error {
 }
 
 func (p *processor) Close() error {
+	log.Info("processor closing ...", zap.String("changefeedID", p.changefeedID))
```

Comment:
```suggestion
	log.Info("processor closing ...", zap.String("changefeed", p.changefeedID))
```

Try `grep 'zap.String("changefeedID"'`